text/0107-construct-library-module-lifecycle.md

```diff
@@ -36,28 +38,22 @@ as possible, and create tracking artifacts that allow us to capture the data req
 to add L2 resources in the future.
 
 - **Stage 0 activities:**
-  - **Automatically publish package** - a package for the module is autogenerated and published in the aws-cdk GitHub
+  - **Automatically publish package** - A package for the module is autogenerated and published in the aws-cdk GitHub
     repo
 
-    - in the module's `package.json` set "Stability": "Stable" to indicate the CloudFormation resource constructs are
-      stable and supported by AWS  
-  - **Automatically publish README** - the module has a rudimentary README doc that (this is an automated activity):
+  - **Automatically publish README** - The module has a rudimentary README doc that (this is an automated activity):
     - links to the relevant CloudFormation docs
     - includes the basic import statement
 
-  - **Create GitHub tracking issue** - create a GitHub tracking issue for the module. The primary role of the tracking
+  - **Create GitHub tracking issue** - Create a GitHub tracking issue for the module. The primary role of the tracking
     issue is to:
     - provide an artifact that represents an entire AWS service that customers can +1 to help us prioritize, and
     - provide a single artifact to place on the public roadmap to indicate work in progress.
-    > **Automation Opportunity:** When a new module is automatically added to the AWS Construct Library, also
-    > auto-generate the module's tracking issue so that it can be assigned an owner as part of the Open Source Developer
-    > Support (OSDS) team's issue triage process.
```

Comment:
Do we have a tracking issue for this?

Reply:
There is now: https://github.com/aws/cdk-ops/issues/538